core/domain/user_services_test.py

```diff
@@ -132,6 +132,41 @@ def test_get_user_id_from_username(self):
         self.assertIsNone(
             user_services.get_user_id_from_username('fakeUsername'))
 
+    def test_set_and_get_user_email_preferences(self):
+        user_id = 'someUser'
+        username = 'username'
+        user_email = 'user@example.com'
+
+        user_services.get_or_create_user(user_id, user_email)
+        user_services.set_username(user_id, username)
+
+        # When UserEmailPreferencesModel is yet to be created,
+        # the value returned by get_email_preferences() should be True.
```

Comment:
Actually -- on second thoughts, the comment here can just be "The UserEmailPreferencesModel is not yet created." It's quite easy to infer what's going on, given that context.